build.gradle

```diff
@@ -24,8 +25,10 @@ ext {
   configBuildToolsVersion = '23'
   configTargetSdkVersion = 23
 
-  support = [appCompat  : 'com.android.support:appcompat-v7:23.0.0',
-             annotations: 'com.android.support:support-annotations:23.0.0']
+  supportVersion = '23.0.0'
```

Comment:
I mean AS fabric plugin. When you add it to the project you cannot build the project when using dependencies with versions set as e.g. $supportVersion. Can you check and confirm?